pkgs/by-name/ai/airi-tamagotchi/package.nix

```diff
@@ -0,0 +1,171 @@
+{
+  lib,
+  stdenvNoCC,
+  rustPlatform,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  cargo-tauri,
+  cudaPackages,
+  pkg-config,
+  pnpm,
+  wrapGAppsHook3,
+  alsa-lib,
+  atk,
+  cacert,
+  darwin,
+  glib,
+  libayatana-appindicator,
+  nodejs,
+  onnxruntime,
+  openssl,
+  protobuf,
+  webkitgtk_4_1,
+}:
+
+rustPlatform.buildRustPackage (final: {
+  pname = "airi-tamagotchi";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "moeru-ai";
+    repo = "airi";
+    rev = "0e907295ab1dc9dc717885a598b8011d9fe76619";
```

Comment:
Why not just use `refs/tags/v${version}`?

Reply:
I synced with the airi flake version of the package and accidentally changed the rev

but since there are so many changes between 0.6.1 and main I'll wait for the next release